content/influxdb/v2.2/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,103 @@ menu:
 weight: 101
 ---
 
+## v2.2 [2022-02-17]
```

Comment:
I don't see any of these, from the changelog:

1. [a40e12b](https://github.com/influxdata/influxdb/commit/a40e12b): Allow changing a password with `influxd recovery user update`
1. [b02c89e](https://github.com/influxdata/influxdb/commit/b02c89e): Option to log flux queries cancelled because of server shutdown
1. [3ec5a57](https://github.com/influxdata/influxdb/commit/3ec5a57): Tell browser about cookie expiry
1. [df01d93](https://github.com/influxdata/influxdb/commit/df01d93): Allow flux http calls to be unlimited
1. [49ce57c](https://github.com/influxdata/influxdb/commit/49ce57c): Remove telegraf endpoint pagination

Reply:
1. Added "Recover user credentials" on line 93.  Will also add docs issue, and send docs PR review separately.
2. Looks like line 36 - do we say somewhere how to view this log?
3. Is line 136 OK?
4. Is line137 OK?
5. Is line 138 OK?

cc @samhld

Reply:
@lesam, please let us know if you have any further edits to this PR for Monday. @samhld has approved, so if nothing further, will merge this PR into our base branch and call it good.

Reply:
thank you!